app/views/estimate_flow/property_entry.html.slim

```diff
@@ -8,7 +8,6 @@
           i18n_key: "property_entry",
           show_smod: true,
           show_joint_ownership: @estimate.partner.present?,
-          show_mortgage: @estimate.property_owned == "with_mortgage",
-          hint: t("estimate_flow.property_entry.second_caption_#{@estimate.level_of_help}")
```

Comment:
Now that neither of the views that renders the `app/views/estimate_flow/forms/_property_entry.html.slim` partial uses the "hint" param any more, I think it would be a good idea to remove mention of that param from the partial, so keep things tidy.

Reply:
I think I have implemented your feedback @patrick-laa , let me know if I've missed something!